extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -95,7 +101,9 @@ const Track = memo(
 			inlineStyle.color = colors.secondary.custom;
 		}
 
-		/* translators: This needs to be a single word with no spaces. It describes the current item in the group. A screen reader will announce it as "[title], current track" */
+		/* translators: This needs to be a single word with no spaces. It describes
+		the current item in the group. A screen reader will announce it as "[title],
+		current track". */
```

Comment:
```suggestion
		/*
		 * translators: This needs to be a single word with no spaces. It describes
		 * the current item in the group. A screen reader will announce it as "[title],
		 * current track".
		 */
```

Reply:
This won't be picked up (see https://github.com/Automattic/jetpack/pull/15281#discussion_r404743899)